source/ceylon/io/base64/base64.ceylon

```diff
@@ -170,3 +177,25 @@ object base64Url
              'w', 'x', 'y', 'z', '0', '1', '2', '3',
              '4', '5', '6', '7', '8', '9', '-', '_'];
 }
+
+
+object base64Mime
+	extends AbstractBase64() {
+	table = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H',
+	'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P',
+	'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X',
+	'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f',
+	'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n',
+	'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
+	'w', 'x', 'y', 'z', '0', '1', '2', '3',
+	'4', '5', '6', '7', '8', '9', '+', '/'];
+		
+	void encodeBytesToChars(ByteBuffer input, ByteBuffer encoded){
+			super.encodeBytesToChars(input, encoded);
+			if (((input.size - input.available) % 57) == 0){
```

Comment:
This part is not tested at all, or have I missed it? It's also not clear to me what this does.